src/Sentry.Maui/Internal/ScreenshotAttachment.cs

```diff
@@ -0,0 +1,52 @@
+namespace Sentry.Maui.Internal;
+
+internal class ScreenshotAttachment : Attachment
+{
+    public ScreenshotAttachment()
+        : this(
+            AttachmentType.Default,
+            new ScreenshotAttachmentContent(),
+            "screenshot.jpg",
+            "image/jpeg")
+    {
+    }
+
+    private ScreenshotAttachment(
+        AttachmentType type,
+        IAttachmentContent content,
+        string fileName,
+        string? contentType)
+        : base(type, content, fileName, contentType)
+    {
+    }
+}
+
+internal class ScreenshotAttachmentContent : IAttachmentContent
+{
+    public Stream GetStream()
+    {
+        try
+        {
+#if !WINDOWS
```

Comment:
So this won't compile it on `net8.0-windows` targets because it deadlocks, right?
This is very useful context worth adding here:

```suggestion
// Not including this on Windows specific build because on WinUI this can deadlock.
#if !WINDOWS
```

Reply:
interesting, it has the same behaviour as the old UWP